data-raw/regions.R

```diff
@@ -0,0 +1,34 @@
+# Get a list of regions and codes from ONS API
+
```

Comment:
Again, I'd quite like to see `year <- 2023` at the top here and the rest of the instances derived from that.

Reply:
Done